content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -173,7 +173,7 @@ data:
 ```
 
 The ConfigMap is created in the `kube-system` namespace because this
-ConfigMap configures a Kubelet, which is Kubernetes system component.
+ConfigMap configures a kubelet, which is Kubernetes system component.
```

Comment:
nit: Lines 175 - 176 are hard to parse. Possible edit:
The ConfigMap, from the previous example, is created in the `kube-system` namespace.
The `kube-system` namespace is used because the kubelet is a Kubernetes system component.

Reply:
Reworded.